WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -136,7 +136,13 @@ public void onResponse(JSONObject response) {
                         mRemoteSettings.localTableId = mBlog.getRemoteBlogId();
                         deserializeDotComRestResponse(mBlog, response);
                         if (!mRemoteSettings.equals(mSettings)) {
+                            // postFormats setting is not returned by this api call so copy it over
```

Comment:
The postFormats is fetched by the `fetchPostFormats` call which happens after the `fetchRemoteData` call. See https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java#L662.

I could expand the comment if that's what you mean @maxme , no prob.